workflows/temporal/temporal.go

```diff
@@ -58,6 +59,9 @@ func (c *TemporalWF) Init(metadata workflows.Metadata) error {
 	if m.Identity != "" {
 		cOpt.Identity = m.Identity
 	}
+	if m.Namespace != "" {
+		cOpt.Namespace = m.Namespace
```

Comment:
Do we need to explicitly specify a default, or will it default to 'default'? ... That's a lotta defaults

Reply:
By default, it will default to 'default': https://pkg.go.dev/go.temporal.io/sdk@v1.17.0/client#pkg-constants

Reply:
So `parseMetadata` should actually be using `contrib/metadata`'s `DecodeMetadata` function which takes the `metadata.Properties` from `Init` and unmarshals them into the struct.

The default value in the struct will be the empty string.

Please make sure that when passing an empty string it defaults to whatever you need it to be. Otherwise you do indeed need to assign defaults!